README.rst

```diff
@@ -348,120 +356,114 @@ For example:
 Building an application
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To build your application locally (for local testing), say this in any directory:
+To build your application locally (for local testing), run this in any directory:
 
 .. code-block:: bash
 
     cartridge build [PATH] [flags]
 
-.. // Please, update the doc in cli/commands on updating this section
+.. // Please update the doc in cli/commands on updating this section
 
-This command requires one argument — the path to your application directory
-(i.e. to the build source). The default path is ``.`` (the current directory).
+The command requires one argument – the path to your application directory
+(that is, to the build source). The default path is ``.`` (the current directory).
```

Comment:
Whenever you can, start new sentences in the source files from a new line. It makes sources better readable. 
See https://www.tarantool.io/en/doc/latest/contributing/docs/markup/intro/#wrapping-text.
```suggestion
The command requires one argument – the path to your application directory
(that is, to the build source).
The default path is ``.`` (the current directory).
```